.github/workflows/publish-maven-central.yml

```diff
@@ -0,0 +1,38 @@
+# This is a manually triggered workflow (hence on: workflow_dispatch).
+# See: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#manual-events
+
+name: Publish to Maven Central
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version, without v prefix, appended to v to locate tag'
+        required: true
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: v${{ github.event.inputs.version }}
+
+      # Gradle 7 requires Java 11 to run
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      - name: Publish to Repositories
+        env:
+          ORG_GRADLE_PROJECT_OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          ORG_GRADLE_PROJECT_OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+          ORG_GRADLE_PROJECT_SIGNING_KEY_ID: ${{ secrets.SIGNING_KEY_ID }}
+          ORG_GRADLE_PROJECT_SIGNING_PASSWORD: ${{ secrets.SIGNING_KEY_PASSWORD }}
+          ORG_GRADLE_PROJECT_SIGNING_KEY_BASE64: ${{ secrets.SIGNING_KEY_RING_FILE_BASE64 }}
```

Comment:
@QuintinWillison could you please add those secrets to the Github repository? :bow:

Reply:
I'm pretty sure these are going to be the same secrets [we already use to publish `ably/ably-asset-tracking-android`](https://github.com/ably/ably-asset-tracking-android/blob/a34b6f137a60a21a28af88d48cef6baa015217eb/.github/workflows/publish-maven-central.yml#L30), so I'm going to add them at `ably` org-level, then remove the repository-level secrets from there. This will make it easier to rotate and otherwise manage them in future, if required. FYI, @lmars @stmoreau 

Our org-level secrets are explicitly shared with just the repositories that need them.

Reply:
As org-level secrets it's appropriate to make their naming more specific, so I'm going to change those with prefix `SIGNING_` to prefix `MAVEN_SIGNING_`. @KacperKluka, I'll push a commit to this PR to make that naming change here also, once I've created the org-level secrets.

Reply:
The org-level secrets have all been created, and the workflow naming changed in https://github.com/ably/ably-java/pull/791/commits/b6eecfe6d79ec14bfc43be024e600e0cc05c4591.

Reply:
Oops, and: https://github.com/ably/ably-java/pull/791/commits/8daa24bf1e3f7d7a006bf2028b998a221245461b (subsequent to https://github.com/ably/ably-java/pull/791/commits/986592b8115c33cc4263be8599705acbefded608).

Reply:
@KacperKluka Could you resolve this conversation if you're happy that this looks correct, alongside the state presented by this PR. Current secrets configuration for this repository:

<img width="792" alt="Screenshot 2022-05-27 at 10 52 11" src="https://user-images.githubusercontent.com/8318344/170676306-54b08807-3635-4554-b2bb-4a4276d17578.png">